<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PStackShow</class>
 <widget class="QDialog" name="PStackShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_s2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Stack 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Go</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lneExp">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_exp">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Expression</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lneS1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>97</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lneS2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>118</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_s1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Stack 1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lneS3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_s3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Stack 3</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
